app/models/miq_alert.rb

```diff
@@ -210,6 +210,14 @@ def add_status_post_evaluate(target, result, event)
     miq_alert_statuses << status
   end
 
+  def find_status(target, event, ems_ref)
+    if event.event_type == "datawarehouse_alert"
```

Comment:
@moolitayer @gtanzillo any reason for this being specific just for the `datawarehouse` alerts?
If indeed that's the case (and I really hope that instead we can extend this to all other alerts), then I would like to find a better solution than an `if` based on the `event_type` string.